.travis.yml

```diff
@@ -13,9 +13,6 @@ git:
   depth: 99999999
 before_install:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
```

Comment:
I think this (and the git depth option) are unnecessary now? If not, it appears we can use `  depth: false` now.

Reply:
Yep, it looks like those are in the default now. I'll remove them.